### PR TITLE
Fix some incorrect links in docs (#28191)

### PR DESCRIPTION
**Pull request by yp05327**
Backport https://github.com/go-gitea/gitea/pull/28191

### CODE REVIEW

File: docs/content/help/faq.en-us.md [hunk 1]
```diff
@@ -39,7 +39,6 @@ If a bug fix is targeted on 1.20.1 but 1.20.1 is not released yet, you can get t
 
 To migrate from Gogs to Gitea:
 
-- [Gogs version 0.9.146 or less](installation/upgrade-from-gogs.md)
```
Comment (nanguanlin6) [inline docs/content/help/faq.en-us.md]: This looks wrong to me. The document was removed in main and its milestone is 1.22. So this backport and the backport to 1.21(though that is merged) should not include this one. If you want to raise a PR to fix the `../../` error, this one can be fixed in 1.21 by the way.

File: docs/content/help/faq.zh-cn.md [hunk 2]
```diff
@@ -41,7 +41,6 @@ menu:
 
 要从Gogs迁移到Gitea：
 
-- [Gogs版本0.9.146或更低](installation/upgrade-from-gogs.md)
```
Comment (nanguanlin6) [inline docs/content/help/faq.zh-cn.md]: .

File: docs/content/help/faq.en-us.md (hunk 1 above)
Comment (yp05327) [reply docs/content/help/faq.en-us.md]: Reverted.

File: docs/content/help/faq.zh-cn.md (hunk 2 above)
Comment (yp05327) [reply docs/content/help/faq.zh-cn.md]: Reverted.

File: docs/content/help/faq.en-us.md (hunk 1 above)
Comment (yp05327) [reply docs/content/help/faq.en-us.md]: > So this backport and the backport to 1.21(though that is merged) should not include this one.

#28235

**Review by delvh — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**